README.md

```diff
@@ -39,13 +39,13 @@ channel automatically.
 
 1. Select "Start Recording" to start audio recording
 
-1. Post the agenda link in the meeting chat box
+1. Post the agenda link (if non-standing) in the meeting chat box
```

Comment:
I believe this should always be posted, as it triggers output on the HTML side when the meeting minutes are parsed. If we don't add this, no agenda will appear here: https://w3c-ccg.github.io/meetings/2022-08-23-traceability/

Reply:
OK. Then the standing agenda should be found at the URI in the instructions (which it is not, today). Better, an agenda should be sent out to the mailing list _before each and every meeting_, whether or not it's the "standing" agenda, and a link to *that message* should be posted to the chat at the start of the meeting.

Reply:
Agreed. I've added a suggested change to fix the URL for the existing process. I see what you mean regarding sending the agenda to the mailing list, and have [opened a separate issue](https://github.com/w3c-ccg/traceability-interop/issues/371) to address that.

Reply:
```suggestion
1. Post the link to today's agenda in the meeting chat box. The link below leads 
   to the default "standing" agenda, and should be posted only if a date-specific 
   agenda was not sent to the mailing list.
```